doc/nestml_language/neurons_in_nestml.rst

```diff
@@ -260,13 +260,13 @@ In order to model an absolute refractory state, in which the neuron cannot fire
    update:
        if is_refractory:
            # neuron is absolute refractory, do not evolve V_m
-           refr_t -= resolution()
```

Comment:
Shouldn't the entire section "Implementing refractoriness" be updated to explain the new way of handling refractoriness?